packages/react-i18n/src/utilities/translate.tsx

```diff
@@ -18,15 +18,18 @@ const PLURALIZATION_KEY_NAME = 'count';
 const SEPARATOR = '.';
 
 const numberFormats = new Map();
-export const memoizedNumberFormatter = function(locale, options = {}) {
+export function memoizedNumberFormatter(
+  locale: string,
+  options: Intl.NumberFormatOptions = {},
+) {
```

Comment:
I was mostly just adding types here, but this seemed like a natural adjustment to make. couldn't think of why we would want a const declaration over a function declaration.